packages/apollo-cli/src/generate.ts

```diff
@@ -1,4 +1,4 @@
-import * as fs from 'fs';
+import { fs } from "apollo-codegen-core/lib/localfs";
```

Comment:
should we just export `fs` from codegen core? I could see us allowing a browser version as well one day

Reply:
This is mostly for internal use, so I didn't export it. Plus, I don't think the current setup would actually allow for a browser version through `localfs`..